src/main/scala/com/ltonetwork/database/package.scala

```diff
@@ -62,8 +62,8 @@ package object database {
 
     while (b.remaining() > 0) {
       val buffer = b.get() match {
-        case crypto.DigestLength    => new Array[Byte](crypto.DigestLength)
-        case crypto.SignatureLength => new Array[Byte](crypto.SignatureLength)
+        case crypto.`digestLength`    => new Array[Byte](crypto.digestLength)
```

Comment:
Any idea why Intellij added those ` symbols when I refactored it? And how do they translate in Scala?